Formula/strace.rb

```diff
@@ -17,11 +17,14 @@ class Strace < Formula
   depends_on "linux-headers"
 
   def install
+    args = %W[
+      --prefix=#{prefix}
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --enable-mpers=check
+    ]
```

Comment:
No need to use `args`. Please leave it as it was, inline.

Reply:
I sort of disagree but also don't care that much at the same time, so ok! :)